src/components/Explore/Explore.tsx

```diff
@@ -287,7 +290,9 @@ const Explore: React.FunctionComponent<{
       ORIGINAL_EXPLORE_METRICS.includes(currentMetric) && numLocations === 1;
 
     // menu labels for metric, time period, and selected locations:
-    const metricMenuLabel = metricLabels[currentMetric];
+    const selectedMetricIndex = EXPLORE_METRICS.indexOf(currentMetric);
+    assert(selectedMetricIndex >= 0);
+    const metricMenuLabel = metricLabels[selectedMetricIndex];
```

Comment:
To find the right menu index to select, we have to look up where `currentMetric` is in the list.